modules/apps/translation/translation-web/src/main/resources/META-INF/resources/translate.jsp

```diff
@@ -136,56 +140,90 @@ renderResponse.setTitle(translateDisplayContext.getTitle());
 								boolean multiline = translateDisplayContext.getBooleanValue(infoField, TextInfoFieldType.MULTILINE);
 							%>
 
-								<clay:row>
-									<clay:col
-										md="6"
-									>
-
-										<%
-										String sourceContent = translateDisplayContext.getSourceStringValue(infoField, translateDisplayContext.getSourceLocale());
-										String sourceContentDir = LanguageUtil.get(translateDisplayContext.getSourceLocale(), "lang.dir");
-										%>
-
-										<c:choose>
-											<c:when test="<%= html %>">
-												<label class="control-label">
-													<%= label %>
-												</label>
-
-												<div class="translation-editor-preview" dir="<%= sourceContentDir %>">
-													<%= sourceContent %>
-												</div>
-											</c:when>
-											<c:otherwise>
-												<aui:input dir="<%= sourceContentDir %>" label="<%= label %>" name="<%= label %>" readonly="true" tabIndex="-1" type='<%= multiline ? "textarea" : "text" %>' value="<%= sourceContent %>" />
-											</c:otherwise>
-										</c:choose>
-									</clay:col>
-
-									<clay:col
-										md="6"
-									>
-
-										<%
-										String targetContent = translateDisplayContext.getTargetStringValue(infoField, translateDisplayContext.getTargetLocale());
-										%>
-
-										<c:choose>
-											<c:when test="<%= html %>">
-												<label class="control-label">
-													<%= label %>
-												</label>
-
-												<div class="translation-editor-preview" dir="<%= LanguageUtil.get(translateDisplayContext.getTargetLocale(), "lang.dir") %>">
-													<%= targetContent %>
-												</div>
-											</c:when>
-											<c:otherwise>
-												<aui:input dir='<%= LanguageUtil.get(translateDisplayContext.getTargetLocale(), "lang.dir") %>' disabled="<%= true %>" label="<%= label %>" name='<%= "infoField--" + infoField.getName() + "--" %>' type='<%= multiline ? "textarea" : "text" %>' value="<%= targetContent %>" />
-											</c:otherwise>
-										</c:choose>
-									</clay:col>
-								</clay:row>
+								<liferay-util:buffer
+									var="ColContentBuffer"
```

Comment:
AFAIK this should be lower camelCase: `colContentBuffer`. Anyway I think we could find a better name for this that represent what it's the content, not where is going to be rendered.

Reply:
Thanks, you are right. It's not even a column but a row. Maybe `contentRowBuffer` fits better?

Reply:
Again we're describing where is going to be rendered (a row), not what. And I don't think `buffer` really needs yo be included in the name.

Reply:
Only `translateRow` comes to mind but is more of the same thing. If I try to put something like `field`, `preview` or ` editor` it seems like I mean smaller components. 
Do you have any specific suggestions?

Reply:
Go with `translationContent`!